server-setup/tasks-handlers.ts

```diff
@@ -12,10 +18,38 @@ const handlers = {
     console.log(
       `📤 Uploading definition for FHIR version ${server.version}...`
     );
+
+    const client: FhirClient = new FetchFhirClient({
+      baseUrl: server.url,
+    });
+
+    if (
+      await isTerminologySourceImported(client, {
+        source: "definitions",
+        system: "definitions",
+        version: "r4",
+      })
+    ) {
```

Comment:
Could you abstract away the client usage in the getter/setter functions 🙏

Reply:
✅